### PR TITLE
add assignment scripts for UAI

### PR DESCRIPTION
**Pull request by mspector**
requires openreview-py version 0.7.4

### CODE REVIEW

File: venues/auai.org/UAI/2018/python/assign-all-areachairs.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    papers = client.get_notes(invitation='auai.org/UAI/2018/-/Blind_Submission')
+    papers_by_forum = {n.forum: n for n in papers}
+
+    for assignment in ac_assignments:
+        assignment_entries = assignment.content['assignedGroups']
+
+        paper = papers_by_forum[assignment.forum]
+
+        for entry in assignment_entries:
+            entry['userId']
```
Comment (melisabok) [inline venues/auai.org/UAI/2018/python/assign-all-areachairs.py]: what does it mean this line?

File: venues/auai.org/UAI/2018/python/assign-all-areachairs.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+            openreview.tools.assign(client, paper.number, 'auai.org/UAI/2018',
+                parent_group_params = {},
+                reviewer_to_add = entry['userId'].encode('utf-8'),
```
Comment (melisabok) [inline venues/auai.org/UAI/2018/python/assign-all-areachairs.py]: reviewer_to_add accepts an email or id?

File: venues/auai.org/UAI/2018/python/assign-all-areachairs.py (hunk 1 above)
Comment (mspector) [reply venues/auai.org/UAI/2018/python/assign-all-areachairs.py]: oops, that must be a mistake, sorry about that

File: venues/auai.org/UAI/2018/python/assign-all-areachairs.py (hunk 2 above)
Comment (mspector) [reply venues/auai.org/UAI/2018/python/assign-all-areachairs.py]: yes

**Comment by mspector** [comment]
closing this in favor of:

https://github.com/iesl/openreview-scripts/pull/664

**Pull request closed without merging**